elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -128,7 +124,7 @@ protected RequestScope(
             JsonApiMapper mapper,
             Logger logger) {
         this(null, transaction, user, dictionary, mapper, logger);
-        this.deferredChecks = new LinkedHashSet<>();
+        this.commitChecks = new LinkedHashSet<>();
```

Comment:
Why do we store commitChecks both here and inside PermissionManager?  Can we just have one location?